<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1288</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_5">
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>100</y>
          <width>261</width>
          <height>521</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pb_createPerson">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Person hinzufügen</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_createCompany">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Unternehmen hizufügen</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_createActivity">
           <property name="text">
            <string>Leistung hinzufügen</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_createCommunication">
           <property name="text">
            <string>Kommunikation hinzufügen</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>120</y>
          <width>191</width>
          <height>491</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pb_seeTables">
           <property name="text">
            <string>Tabellen ansehen</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_modifyTables">
           <property name="text">
            <string>Einträge bearbeiten</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="l_db_status">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>650</y>
          <width>180</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Keine Datenbank geöffnet</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_6">
       <widget class="QLabel" name="l_title">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>260</y>
          <width>551</width>
          <height>111</height>
         </rect>
        </property>
        <property name="text">
         <string>GLR Sponsorendatenbank</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_password">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>390</y>
          <width>341</width>
          <height>41</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Passwort</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1288</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="openDatabase"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openDatabase">
   <property name="text">
    <string>Datenbank öffnen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
